### ISSUE TITLE
react-virtual-scroller not available

### ISSUE THREAD

**Issue by stevenpietzsch** [zippyui/react-datagrid#260]
Hello, 
I was wondering about the [react-vitual-scroller](https://github.com/zippyui/react-virtual-scroller) dependency. Obviously this repo ist not available anymore. Is the scroller no longer supported?

**Comment by joshjung** [comment]
Yes please! I need to make a PR on this repository as soon as possible please.
